config/locales/en/admin/whats_new.yml

```diff
@@ -4,14 +4,25 @@ en:
       title: What’s new in Whitehall Publisher
       summary: |
         Summary of updates to Whitehall Publisher, content design guidance, or the design of GOV.UK.
-      last_updated: Last updated 5 September 2022
+      last_updated: Last updated 3 October 2022
       introduction:
         heading: Moving to the Design System
         body_govspeak: |
           Whitehall Publisher will be moving from the current styling to use the styling and components of the [GOV.UK Design System](https://design-system.service.gov.uk/).
           This will make Whitehall Publisher more accessible, more user friendly and give you a consistent user experience throughout your publishing journey on GOV.UK.
 
           [Read more about the changes on the content community Basecamp](https://3.basecamp.com/4322319/buckets/15005645/messages/5287245970).
+      upcoming_changes:
+        heading: Upcoming changes
+        updates:
+          - heading: Topic taxonomy tags page moves to the Design System
+            area: Creating and updating documents
+            type: improved
```

Comment:
I may be over thinking it so this is a bit nitpicky, but I wonder if past tense tags aren't appropriate for upcoming changes. I'd wonder if "improvement" would be better. 

May be good to make all the tags present tense:

Improved => Improvement
Fixed => Fix
Changed => Change

Reply:
Great stuff. I unresolved this as @DilwoarH will need to change the code slightly to accomodate them.

Reply:
Oh gosh, sorry I see you fixed the code! Great job @johnpauldickie